pkg/encoding/v1/encoding.go

```diff
@@ -219,11 +222,83 @@ func (m *Mount) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return nil
 }
 
+type Health struct {
+	// Data holder for ReadinessProbe while parsing
```

Comment:
I'm still missing WHY part of this.
We have to explain why we are doing it like this via ` interface{}`.
We will forget why it was done like this next time we are reading this.

I already forgot it and you explained it to me last week :-D